docs/iidm/exporter/iidm.md

```diff
@@ -119,7 +119,8 @@ By default all extensions will be exported.
 ## iidm.export.xml.skip-extensions
 The `iidm.export.xml.skip-extensions` property is an optional property that defines whether the XIIDM exporter skips exporting the 
 network extensions or not. Its default value is `false`.
-This property is deprecated since v2.4.0. Use the `iidm.export.xml.export-mode` property instead.
+This property is deprecated since v2.4.0. Set the `iidm.export.xml.extensions` to an empty string instead.
```

Comment:
It is a bit confusing to read that it **is** deprecated and then that it has been removed... Maybe phrase it like this:
> This property has been deprecated since v2.4.0 before being removed in v3.3.0. Set... etc.